sparse/_utils.py

```diff
@@ -79,6 +79,7 @@ def _zero_of_dtype(dtype):
 def random(
     shape,
     density=0.01,
```

Comment:
```suggestion
    density=None,
```

Reply:
Wouldn't this break backwards compatibility?

Reply:
Wouldn't this break backwards compatibility if neither `density` nor `nnz` are provided?

Reply:
See https://github.com/pydata/sparse/pull/410#issuecomment-696813370